src/gui/gui_graphics.hpp

```diff
@@ -262,27 +273,27 @@ struct element_data {
 
 	static constexpr uint8_t ex_is_top_level = 0x01;
 
-	xy_pair position;		 // 4
-	xy_pair size;				 // 8
-	dcon::text_key name; // 12
+	xy_pair position; // 4bytes
+	xy_pair size; // 8bytes
+	dcon::text_key name; // 12bytes
 
 	union alignas(4) internal_data {
-		text_base_data text_common;		// +5
-		button_data button;						// +5 + ? +3
-		text_data text;								// +5 + ? +4
-		image_data image;							// +6
+		text_base_data text_common; // +5
+		button_data button; // +5 + ? +3
+		text_data text; // +5 + ? +4
+		image_data image; // +6
 		overlapping_data overlapping; //+5
-		list_box_data list_box;				// +11
-		scrollbar_data scrollbar;			//+10
-		window_data window;						// +4
-		position_data position;				//+0
+		list_box_data list_box; // +11
+		scrollbar_data scrollbar; //+10
+		window_data window; // +4
+		position_data position; //+0
 
 		internal_data() {
```

Comment:
and this is the union that needs to be zeroed. Doing it in the constructor should be fine